lib/runner/cli/cli.js

```diff
@@ -177,6 +178,20 @@ class CliRunner {
     this.createTestRunner();
     this.setupConcurrency();
 
+    analyticsCollector.updateSettings(this.test_settings);
+    analyticsCollector.updateLogger(Logger);
+
+    analyticsCollector.event('nw_test_run', {
+      arg_env: this.argv.env,
+      arg_parallel: this.argv.parallel,
+      browserName: this.test_settings.desiredCapabilities.browserName,
+      testWorkersEnabled: this.test_settings.testWorkersEnabled,
+      use_xpath: this.test_settings.use_xpath,
+      isBstack: this.test_settings.desiredCapabilities['bstack:options'] !== undefined,
+      test_runner: JSON.stringify(this.test_settings.test_runner)
```

Comment:
The only relevant thing here would be to collect the test_runner "type".

Reply:
updated.